### DISCUSSION TITLE
v1.1.11

### DISCUSSION THREAD

**Discussion by coreybutler** [coreybutler/nvm-windows#961] [General]
This is a maintenance release. A new experimental `debug` command has been added for troubleshooting.

<img width="590" alt="image" src="https://user-images.githubusercontent.com/770982/231642406-aafee810-7e5b-40ae-813a-74e536b1fde6.png">

## What's Changed
* Update reference links, add Google source by @mon-jai in https://github.com/coreybutler/nvm-windows/pull/909
* Fix install.cmd PATH bug by @davidribyrne in https://github.com/coreybutler/nvm-windows/pull/918
* Add Winget Releaser workflow & Dependabot by @sitiom in https://github.com/coreybutler/nvm-windows/pull/885
* Bump actions/stale from 4 to 7 by @dependabot in https://github.com/coreybutler/nvm-windows/pull/921
* Bump vedantmgoyal2009/winget-releaser from 1 to 2 by @dependabot in https://github.com/coreybutler/nvm-windows/pull/928
* Installer improvements when alt creds are required for elevation by @noelhibbard in https://github.com/coreybutler/nvm-windows/pull/459
* Avoid panic after connection failure by @qiulaidongfeng in https://github.com/coreybutler/nvm-windows/pull/949
* Bump actions/stale from 7 to 8 by @dependabot in https://github.com/coreybutler/nvm-windows/pull/952
* Bump golang.org/x/sys from 0.0.0-20190813064441-fde4db37ae7a to 0.1.0 in /src by @dependabot in https://github.com/coreybutler/nvm-windows/pull/936
* Bump golang.org/x/text from 0.3.2 to 0.3.8 in /src by @dependabot in https://github.com/coreybutler/nvm-windows/pull/935

## New Contributors
* @mon-jai made their first contribution in https://github.com/coreybutler/nvm-windows/pull/909
* @davidribyrne made their first contribution in https://github.com/coreybutler/nvm-windows/pull/918
* @sitiom made their first contribution in https://github.com/coreybutler/nvm-windows/pull/885
* @dependabot made their first contribution in https://github.com/coreybutler/nvm-windows/pull/921
* @noelhibbard made their first contribution in https://github.com/coreybutler/nvm-windows/pull/459
* @qiulaidongfeng made their first contribution in https://github.com/coreybutler/nvm-windows/pull/949

**Full Changelog**: https://github.com/coreybutler/nvm-windows/compare/1.1.10...1.1.11

<hr /><em>This discussion was created from the release <a href='https://github.com/coreybutler/nvm-windows/releases/tag/1.1.11'>v1.1.11</a>.</em>

**Comment by sitiom** [comment]
Winget Releaser workflow failed. Please see https://github.com/coreybutler/nvm-windows/pull/885#issuecomment-1506295022

**Comment by vchelamkuri** [comment]
@coreybutler Any plans of making the same release for [nvm.portable](https://community.chocolatey.org/packages/nvm.portable)? It is still at version 1.1.9.

**Reply by sitiom** [reply]
The Chocolatey package is not maintained by him. Ask from the package upstream instead: https://github.com/arieljannai/chocolatey-nvm-windows

**Comment by Fhernd** [comment]
Hello,

I encountered an issue with version 1.1.11 where it failed to change the Node version. To resolve this, I had to downgrade to version 1.1.10.

I hope this information is helpful to someone experiencing the same issue.

Best regards,
John.

**Reply by coreybutler** [reply]
This would be more useful if you mentioned what the error was and why it occurred.

**Reply by Linch3x** [reply]
I succesfully downloaded and installed the nvm-setup.exe but when i try to execute the nodejs.exe nothing happens, i can only see a flashing windows in black trying to open and automatically closes, it last less than a second is a blink of an eye, can't open the nodejs to practice.

**Reply by coreybutler** [reply]
This tool is a version manager, not node itself. You need to open a new terminal window and install/use the version of Node you want before it will work. For example, `nvm install latest`, then `nvm use latest`.

**Comment by israelabedi** [comment]
Why cant I download this .exe file?

**Reply by coreybutler** [reply]
The executables are always available in the releases, i.e. https://github.com/coreybutler/nvm-windows/releases/tag/1.1.11

**Reply by Abdulkbk** [reply]
I have also been trying to install the setup but unable to. I think there might be something wrong

**Reply by coreybutler** [reply]
Browsers and corporate policies often prevent the download of exe files from the internet. This is why the app is also packaged in a zip file. The zip file just contains the exe. Download and extract it.

**Comment by juan-codes45** [comment]
am having issue downloading the exe file
